packages/ui/src/components/Form/__snapshots__/CustomAutoFields.test.tsx.snap

```diff
@@ -607,7 +607,7 @@ exports[`CustomAutoFields renders \`AutoFields\` for common fields 1`] = `
           aria-controls="expandable-section-content"
           aria-expanded="false"
           class="pf-v5-c-expandable-section__toggle"
-          id="expandable-section-toggle"
+          id="expandable-section-toggle-processor consumer (advanced)"
```

Comment:
is it fine to have ids with spaces and parenthesis? (I do no tknow, just noticed that others around do not have that and using dash instead of spaces)

Reply:
From the docs: https://html.spec.whatwg.org/multipage/dom.html#the-id-attribute

> When specified on [HTML elements](https://html.spec.whatwg.org/multipage/infrastructure.html#html-elements), the [id](https://html.spec.whatwg.org/multipage/dom.html#the-id-attribute) attribute value must be unique amongst all the [IDs](https://dom.spec.whatwg.org/#concept-id) in the element's [tree](https://dom.spec.whatwg.org/#concept-tree) and must contain at least one character. The value must not contain any [ASCII whitespace](https://infra.spec.whatwg.org/#ascii-whitespace).

Although the browser is not complaining (neither Chrome nor Firefox), probably it would be better to replace all whitespace with `-`